src/main/java/com/rogerr/custom/model/Product.java

```diff
@@ -0,0 +1,51 @@
+package com.rogerr.custom.model;
+
+public class Product {
+
+	private String id;
+	private String name;
+	private String description;
+	private double price;
+	private String storeOwner;
+	
+	public Product(String id, String name, String description, double price, String storeOwner) {
+		super();
+		this.id = id;
+		this.name = name;
+		this.description = description;
+		this.price = price;
+		this.storeOwner = storeOwner;
+	}
```

Comment:
Avoid custom constructors, unless you really need them. Also IF you define a custom constructor, you need to define the default or constructor, particularly when you attempt to use these POJOS for persistence later.